infrastructure/modules/postgres_database/main.tf

```diff
@@ -0,0 +1,22 @@
+
+
+data "azurerm_resource_group" "postgres_database" {
+  name = var.resource_group_name
+}
+
+data "azurerm_postgresql_flexible_server" "postgres_database" {
+  name                = var.postgres_server_name
+  resource_group_name = data.azurerm_resource_group.postgres_database.name
+}
+
+resource "azurerm_postgresql_flexible_server_database" "postgres_database" {
+  name      = var.database_name
+  server_id = azurerm_postgresql_flexible_server.postgres_database.id
+  collation = "en_US.utf8"
```

Comment:
Kanskje finne ut av hvilken collation vi faktisk skal bruke?

Reply:
jepp, enn så lenge brukes ikke denne modulen av noen. Så at databasene som er i azure nå brukes denne

Reply:
Ah - hvis de basene vi har per naa bruker `en_US` - saa er det greit nok for min del :)